fluentnamer/readme.md

```diff
@@ -19,17 +19,28 @@ pipeline:
     flatten-models: true
     flatten-payloads: true
     naming:
+      preserve-uppercase-max-length: 2
       override:
-        api: Api
-        cpk: Cpk
-        sas: Sas
-        url: Url
-        xml: Xml
+        ip: Ip
+        id: Id
```

Comment:
I've lower it to 2. So all 3 letter acronym will be not be in full uppercase.

https://azure.github.io/azure-sdk/java_design.html#naming-patterns